src/topology/algebra/ordered.lean

```diff
@@ -324,27 +346,27 @@ from le_antisymm
     | _, h, (or.inr rfl) := inf_le_right_of_le $ infi_le_of_le b $ infi_le _ h
     end)
 
-lemma tendsto_orderable {f : β → α} {a : α} {x : filter β} :
-  tendsto f x (𝓝 a) ↔ (∀a'<a, {b | a' < f b} ∈ x) ∧ (∀a'>a, {b | a' > f b} ∈ x) :=
-by simp [nhds_eq_orderable a, tendsto_inf, tendsto_infi, tendsto_principal]
+@[nolint] lemma tendsto_order {f : β → α} {a : α} {x : filter β} :
```

Comment:
Is this `nolint` because of `>`? Does it deserve a comment? Hopefully @fpvandoorn will soon find a way to explain `#lnit` this is not bad and he will need help finding all the `@[nolint]` related to this.

Reply:
We need a linter that reports `nolint`s that are no longer necessary :roll_eyes:

Reply:
I added a `Note` explaining this. This way, if the linter is fixed, then the removable `nolint` attributes will be easy to locate.